src/code/audio/code_8019AF00.c

```diff
@@ -1,6 +1,1880 @@
+            OCARINA_BTN_C_DOWN,
+        },
+    },
+};
+
+// Static In-Function Data
+u8 sScarecrowAfterCreditsState = 0;
+u8 sScarecrowAfterCreditsIntrumentId = 1;
+u16 sScarecrowAfterCreditsTimer = 1200;
+u8 sRequestCustomSequence = false;
+s8 D_801D8B30[] = {
+    0,  2,  4,   6,   8,   10,  12,  14,  16,  18,  20,  22,  24,  25,  26,  27,  28,  29,  30, 31, 32, 33,
+    34, 35, 36,  37,  38,  39,  40,  41,  42,  43,  44,  45,  46,  47,  48,  49,  50,  51,  52, 53, 54, 55,
+    56, 56, 57,  57,  58,  58,  59,  59,  60,  60,  60,  61,  61,  61,  62,  62,  62,  63,  63, 63, 64, 64,
+    64, 65, 65,  65,  66,  66,  66,  67,  67,  67,  68,  68,  69,  69,  70,  70,  71,  71,  72, 73, 74, 75,
+    76, 77, 78,  79,  80,  81,  82,  83,  84,  85,  86,  87,  88,  89,  90,  91,  92,  93,  94, 95, 96, 97,
+    98, 99, 100, 101, 102, 103, 105, 107, 109, 111, 113, 115, 117, 119, 121, 123, 125, 127,
+};
+f32 sBigBellsVolume[8] = {
+    1.0f, 0.9f, 0.8f, 0.7f, 0.6f, 0.5f, 0.4f, 0.3f,
+};
+u8 sBgmPlayers[2] = { SEQ_PLAYER_BGM_MAIN, SEQ_PLAYER_BGM_SUB };
+f32 sSfxOriginalPos[] = { 0.0f, 0.0f, 0.0f };
+
+// OoT's soundEffects from EnRiverSound
+const u16 gAudioEnvironmentalSfx[] = {
+    NA_SE_EV_RIVER_STREAM - SFX_FLAG,   NA_SE_EV_WAVE - SFX_FLAG,
+    NA_SE_EV_WATER_WALL_BIG - SFX_FLAG, NA_SE_EV_WATER_WALL - SFX_FLAG,
+    NA_SE_EV_MAGMA_LEVEL - SFX_FLAG,    NA_SE_EV_MAGMA_LEVEL_M - SFX_FLAG,
+    NA_SE_EV_MAGMA_LEVEL_L - SFX_FLAG,  NA_SE_EV_TORCH - SFX_FLAG,
+    NA_SE_EV_FOUNTAIN - SFX_FLAG,       NA_SE_EV_DRAIN - SFX_FLAG,
+    NA_SE_EV_CROWD - SFX_FLAG,          NA_SE_EV_WATER_CONVECTION - SFX_FLAG,
+    NA_SE_EV_GORON_CHEER - SFX_FLAG,    NA_SE_EV_WATER_WALL_BIG_SILENT - SFX_FLAG,
+    NA_SE_EV_WATER_BUBBLE - SFX_FLAG,   NA_SE_EV_COW_CRY_LV - SFX_FLAG,
+    NA_SE_EV_MAKE_TURRET - SFX_FLAG,    NA_SE_EV_BOILED_WATER_S - SFX_FLAG,
+    NA_SE_EV_BOILED_WATER_L - SFX_FLAG, NA_SE_EV_WAVE_S - SFX_FLAG,
+    NA_SE_EV_WAVE_S - SFX_FLAG,         NA_SE_EV_WAVE_S - SFX_FLAG,
+};
+
+/**
+const u8 sIsOcarinaSongReserved[OCARINA_SONG_MAX] = {
+    true,  // OCARINA_SONG_SONATA
+    true,  // OCARINA_SONG_GORON_LULLABY
+    true,  // OCARINA_SONG_NEW_WAVE
+    true,  // OCARINA_SONG_ELEGY
+    true,  // OCARINA_SONG_OATH
+    false, // OCARINA_SONG_SARIAS
+    true,  // OCARINA_SONG_TIME
+    true,  // OCARINA_SONG_HEALING
+    true,  // OCARINA_SONG_EPONAS
+    true,  // OCARINA_SONG_SOARING
+    true,  // OCARINA_SONG_STORMS
+    false, // OCARINA_SONG_SUNS
+    true,  // OCARINA_SONG_INVERTED_TIME
+    true,  // OCARINA_SONG_DOUBLE_TIME
+    true,  // OCARINA_SONG_GORON_LULLABY_INTRO
+    false, // OCARINA_SONG_WIND_FISH_HUMAN
+    false, // OCARINA_SONG_WIND_FISH_GORON
+    false, // OCARINA_SONG_WIND_FISH_ZORA
+    false, // OCARINA_SONG_WIND_FISH_DEKU
+    true,  // OCARINA_SONG_EVAN_PART1
+    true,  // OCARINA_SONG_EVAN_PART2
+    false, // OCARINA_SONG_ZELDAS_LULLABY
+    true,  // OCARINA_SONG_SCARECROW
+    false, // OCARINA_SONG_TERMINA_WALL
+};
+*/
```

Comment:
What is this and why is this commented out?

Reply:
This is rodata that is extracted and placed at the top of `func_8019C8D8` but belongs at the top of the function. The other rodata, `sAudioOcarinaUnusedTextX` needs to be below `sIsOcarinaSongReserved`, so that's why this data is temporarily below `func_8019C8D8`

https://github.com/engineer124/mm/blob/7ea60a08c675c00140a038df40276216ada2ba9c/src/code/audio/code_8019AF00.c#L1933-L1940

However, once `func_8019C8D8` is decompiled from my future `AudioOcarina` PR, then this data can be uncommented, `sAudioOcarinaUnusedTextX` can return to the top of the file, and the final file will look like this:
https://github.com/engineer124/mm/blob/213f7f7851f37c93c4b67615f14b025b8815159d/src/code/audio/audio_interface.c#L1848-L1908